### PR TITLE
Add MetaLake map

### PR DESCRIPTION
**Pull request by ghost**
:cl: steamp0rt
add: MetaLake map
/:cl:

If i'm missing any Hippie features, tell me. And yes, I realize one of the window pairs in sci-chem doesn't have grilles, I fixed that.

metalake-1

metalake_bar
metalake_maint
metalake_pool
metalake_scichem

metalake_forge

### CODE REVIEW

**Comment by ghost** [comment]
Darn solar flares causing CI builds to fail!

**Comment by praisenarsie** [comment]
Do you have the reagent forge?

**Comment by ghost** [comment]
@praisenarsie Oh, uh, pic?

**Comment by ghost** [comment]
nvm i'll look at it myself

**Comment by ghost** [comment]
Reagent Forge added

**Comment by ghost** [comment]
And yes, I am aware the alarm and warning sign look like they are on top of each other in the reagent forge, but you can only see one at a time in game. Same with "fire warning" sign and the extinguisher cabinet in the firing range

**Comment by ghost** [comment]
Just gonna ping maintainers that i know of

@praisenarsie @JohnGinnane @Pyko1

File: config/maps.txt [hunk 1]
```diff
@@ -17,6 +17,11 @@ map hippiestation
 	#voteweight 1.5
 endmap
 
+map metalake
+	minplayers 20
+	#voteweight 0.5
```
Comment (praisenarsie) [inline config/maps.txt]: eh change this to 1.5 or remove it

**Comment by praisenarsie** [comment]
Testing something

**Comment by praisenarsie** [comment]
I need you to commit something to the map again so MDB actually gives a diff.

**Comment by ghost** [comment]
Done

**Comment by yoyobatty** [comment]
Can you do a map render using dmmtools?

**Comment by praisenarsie** [comment]
@yoyobatty YOYO PLEASE

**Comment by ghost** [comment]
metalake-1

**Comment by ghost** [comment]
It is done, @yoyobatty

**Comment by yoyobatty** [comment]
So this is a hippified version of metastation?

**Comment by praisenarsie** [comment]
no, it's a hippified version of runtimestation

**Comment by praisenarsie** [comment]
the location of the forge is garbage + it isn't surrounded by reinforced walls

**Comment by ghost** [comment]
@praisenarsie It was the only space I could find that met the requirements of 

A. big enough
B. nothing really important there
C. won't bulge out from the station like a tumor

**Comment by ghost** [comment]
Anyways I added rwalls @praisenarsie

**Comment by ghost** [comment]
@Carbonhell What's your discord?

**Comment by Carbonhell** [comment]
Carbonhell#4499
anyway this PR requires atleast 2 experienced mappers to check it carefully with the map editor. I don't want a broken map to get in, so whoever can, check it please.
Wouldn't be against it, but it has to be manually set by an admin, I'm still against map rotation.
+1

**Comment by Carbonhell** [comment]
@HippieStation/maintainers Mapcheck this.

**Comment by Carbonhell** [comment]
http://mapdiffbot.dextraspace.net/Files/84665083/11592525/1/diff.png
to mapcheck more easily

File: config/maps.txt [hunk 2]
```diff
@@ -17,6 +17,10 @@ map hippiestation
 	#voteweight 1.5
 endmap
 
+map metalake
+	minplayers 20
+endmap
```
Comment (praisenarsie) [inline config/maps.txt]: actually you might as well add #voteweight 1.5 since it's a hippie map

File: _maps/metastation.dm [hunk 3]
```diff
@@ -1 +1 @@
-#define FORCE_MAP "_maps/metastation.json"
+#define FORCE_MAP "_maps/metalake.json"
```
Comment (Carbonhell) [inline _maps/metastation.dm]: ??

File: hippiestation/code/modules/paperwork/paper_premade.dm [hunk 4]
```diff
@@ -0,0 +1,3 @@
+/obj/item/paper/fluff/stations/hippie/dontletcarbonseethis
```
Comment (Carbonhell) [inline hippiestation/code/modules/paperwork/paper_premade.dm]: what the fuck

File: tgstation.dme [hunk 5]
```diff
@@ -20,8 +20,6 @@
 #include "code\__DEFINES\_protect.dm"
 #include "code\__DEFINES\_tick.dm"
 #include "code\__DEFINES\access.dm"
-#include "code\__DEFINES\bsql.config.dm"
-#include "code\__DEFINES\bsql.dm"
```
Comment (Carbonhell) [inline tgstation.dme]: what..

**Review by Carbonhell — CHANGES_REQUESTED** [review]

File: tgstation.dme (hunk 5 above)
Comment (ghost) [reply tgstation.dme]: whoops

File: hippiestation/code/modules/paperwork/paper_premade.dm (hunk 4 above)
Comment (ghost) [reply hippiestation/code/modules/paperwork/paper_premade.dm]: @praisenarsie's joke. i'll remove it.

File: _maps/metastation.dm (hunk 3 above)
Comment (ghost) [reply _maps/metastation.dm]: will fix.

**Comment by ghost** [comment]
+1

File: _maps/metastation.dm (hunk 3 above)
Comment (ghost) [reply _maps/metastation.dm]: Fixed

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
>no bots checked this pr
the fuck
well anyway i assume the dmi is good too, everything else is fine and the map really needs to be tested ingame properly
Merging this as disabled, admins will have to force it

**Comment by Carbonhell** [comment]
there's this which kinda gives me a bad feeling
/obj/effect/spawner/lootdrop/armory_contraband{
	loot = list(/obj/item/gun/ballistic/automatic/pistol = 5, /obj/item/gun/ballistic/shotgun/automatic/combat = 5, /obj/item/gun/ballistic/revolver/mateba, /obj/item/gun/ballistic/automatic/pistol/deagle, /obj/item/storage/box/syndie_kit/throwing_weapons = 3)
	},
but if it's a problem it can very easily be removed

**Pull request merged**